Python/ceval.c

```diff
@@ -4618,7 +4615,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
                     goto error;
                 }
                 _PyFrame_SetStackPointer(frame, stack_pointer);
-                frame->f_lasti += INLINE_CACHE_ENTRIES_CALL;
+                frame->next_instr += INLINE_CACHE_ENTRIES_CALL;
```

Comment:
There are a few `frame->next_instr += X` here and below.

Provided `frame->next_instr` is up to date with `next_instr`, this would be more efficient as
```
JUMPBY(X); frame->next_instr = next_instr
``` 
It is possible the compiler can work that out for itself, but I think the `JUMPBY` version is a bit clearer. Maybe add a `SAVE_IP()` macro so this becomes `JUMPBY(X); SAVE_IP();`